### DISCUSSION TITLE
GLIBC_2.32 not found

### DISCUSSION THREAD

**Discussion by No-Github** [projectdiscovery/naabu#696] [Q&A] [answered]
The GLIBC version on the compiling machine is too high, which causes the compiled file to be unable to run on the AWS machine

<img width="1308" alt="image" src="https://user-images.githubusercontent.com/18167071/233918686-7c44382c-a1c3-4472-b42a-f1689f616b15.png">

**Comment by V0WKeep3r** [comment]
same problem.

**Comment by V0WKeep3r** [comment]
I wanna kown which version is no need GLIBC_2.34, because my centOS tried to make and failed twice.

**Comment by idiotboxai** [comment]
same problem

**Comment by xvapourx** [comment]
same issue

**Comment by noob0x01** [comment]
If you are using Ubuntu do:

`sudo apt-get install build-essential`

This solved the problem. It installs the gcc/g++ compilers and libraries.

**Comment by No-Github** [comment]
![Snipaste_2023-05-10_19-07-31](https://github.com/projectdiscovery/naabu/assets/18167071/12989cd2-baae-44f9-b7dd-0575c351aba0)

It doesn't work

**Comment by Mzack9999** [comment, marked as answer]
Unfortunately, naabu is tied to the `GLIBC_2.34` version installed on the GitHub worker box used to build the release binary. Generally, go is quite tolerant regarding the installed GLIBC version, but with naabu, it's tied to a specific version because of dynamic linking with `libpcap.`
The best option would be to install naabu on the box via go install:
```console
$ go install -v github.com/projectdiscovery/naabu/v2/cmd/naabu@latest
```
This requires go + libpcap on the machine.

**Comment by dacade** [comment]
Ubuntu 20.04 can solve it
1、`sudo vi /etc/apt/sources.list`
2、add High version source：
`deb http://th.archive.ubuntu.com/ubuntu jammy main`
3、`sudo apt update && sudo apt install libc6`

**Reply by Shea-Des** [reply]
centos7 系统如何处理

**Reply by MetzinAround** [reply]
![Screenshot -1025 at 15 01](https://github.com/projectdiscovery/naabu/assets/65838556/f5b47b80-9b2a-40f6-af73-f5aaa15c7369)

**Reply by MetzinAround** [reply]
hi! The best way to do this is to follow the marked answer above from Mzack9999. https://github.com/projectdiscovery/naabu/discussions/696#discussioncomment-6061639

**Comment by eiko4** [comment]
+1

Hmm...

**As default naabu is configured with a assumption that you are running it from VPS.**
